functions/Remove-DbaOrphanUser.ps1

```diff
@@ -330,7 +330,7 @@ function Remove-DbaOrphanUser {
 						$User = $null
 					}
 					catch {
-						Write-Message -Level Error -Message $_
+						Stop-Function -Message $_ -Target $db -Continue
```

Comment:
Appending `-ErrorRecord $_` would be good here and removing $_ as the message. You could just say "Failure" if you like

Reply:
Thanks @potatoqualitee ! done.

Reply:
You can actually get that inner exception expanded to a point with Stop-Function by using the `-Exception` parameter. Fred added that months back so you can do:

```
Stop-Function -Message "Whatever failed" -Target <your target variable> -ErrorRecord $_ -Exception $_.Exception -Continue
```

I don't know if you could just pass `$_.Exception.InnerException` as well but not sure how that effects it.